src/components/projects/create-wp-form/create-wp-form/create-wp-form.module.css

```diff
@@ -1,4 +1,4 @@
 /*
  * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
  * See the LICENSE file in the repository root folder for details.
- */
+ */
```

Comment:
This is missing an end of file newline, it's in the new PR checklist, but should also be done automatically by prettier I believe

Reply:
There are also multiple of this I think